src/injections/composeInjections/index.js

```diff
@@ -1,8 +1,4 @@
-import mergeInjections from '../mergeInjections';
-
-function composeInjections(...injections) {
-  const injectionsDescription = mergeInjections(injections);
```

Comment:
Why `mergeInjections` is deleted ? Does this actually support an array of injections ? 🤔
Remember that an `injection` is described as a plain object. What this line does is to ensemble these into a big object.

Reply:
`composeInjections` was always being passed an object with the injections similar to:
```
behaviour: ()=>{},
success: ()=>{},
...
```

But then we used the spread operator to have an array with the first element being that injections object and we used `mergeInjections` to turn that array back into an object.

So `injections` was always the same as `injectionsDescription`

Reply:
Ah didn't saw, the subtle change is at src/middlewares/fetch.js